test/parallel/test-debug-signal-cluster.js

```diff
@@ -29,6 +29,10 @@ let pids;
 
 child.stderr.on('data', function(data) {
   const childStderrOutputString = data.toString();
+  if (/DeprecationWarning: node --debug is deprecated\./
+    .test(childStderrOutputString))
```

Comment:
@Trott as I reviewed I think it might be best to keep the deprecation warning isolated from the existing part of the test so it can be better understood and maintained/removed in the future. So instead I updated to address multi-line incoming data, PTAL and let me know what you think. Thanks!